packages/contracts/contracts/TroveManager.sol

```diff
@@ -477,15 +476,14 @@ contract TroveManager is LiquityBase, Ownable, CheckContract, ITroveManager {
     {
         singleLiquidation.entireTroveDebt = _entireTroveDebt;
         singleLiquidation.entireTroveColl = _entireTroveColl;
-        uint collToOffset = _entireTroveDebt.mul(MCR).div(_price);
+        uint collToLiquidate = _entireTroveDebt.mul(MCR).div(_price);
```

Comment:
`collToLiquidate` here doesn't exclude the gas compensation. 

However in Normal Mode and the branches in Recovery Mode that don't do capped liquidation, `vars.collToLiquidate` does exclude the gas compensation.  Maybe we should make the naming consistent?

Reply:
Oh, I didn’t realize. Yes, sure, let’s change it here then (as it’s easier, it’s a new temp variable that only appears here). Suggestions? :wink: